types/it-pushable/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for it-pushable 1.3
+// Project: https://github.com/alanshaw/it-pushable
+// Definitions by: Junxiao Shi <https://github.com/yoursunny>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.2
+
+/// <reference lib="esnext.asynciterable" />
```

Comment:
Hm, so, I talked to the team about this, and I think we can’t have this here. It will pull those types in for anyone who installs these typings, even if their own lib settings don’t include it. (Granted, it seems unlikely that this is a real problem in practice, since anyone installing these typings _probably_ has a runtime that supports async iterable.) The right thing to do is probably to put this in the tsconfig `lib` section (for some reason I thought that DefinitelyTyped _only_ let you use `es6` and `dom`, but I might be misremembering), or, if that doesn’t work, go back to the `@types/node` reference since Node ships with an AsyncIterable implementation.

Reply:
Moved to `tsconfig.json`. `npm test` passes.